client/sites-dashboard/components/sites-site-url.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from '@emotion/styled';
+import { ReactNode, useState, AnchorHTMLAttributes } from 'react';
+import ExternalLink from 'calypso/components/external-link';
+
+const ExternalSiteLink = styled( ExternalLink )`
+	text-overflow: ellipsis;
+	overflow: hidden;
+	display: inline-block;
+	font-size: 14px;
+	color: var( --studio-gray-60 ) !important;
+
+	&:hover,
+	&:visited {
+		font-weight: 500;
```

Comment:
There seems to be a little bit of a jiggle here:


https://user-images.githubusercontent.com/36432/184236306-6798ebff-7d7f-4ce0-8185-a27e390f77d2.mp4

Is this intentional?

Reply:
I was thinking of an underline, more like the hover style of the site title.

Reply:
Will update to underline